pytorch_lightning/trainer/connectors/logger_connector.py

```diff
@@ -121,13 +121,23 @@ def _log_on_evaluation_epoch_end_metrics(self, eval_results, using_eval_result):
                         flat = {'val_loss': eval_result}
                     else:
                         flat = flatten_dict(eval_result)
```

Comment:
nit: there's this implementation and also one in the lightning loggers base for flattening dicts. it'd be nice to have one utility so they don't diverge

Reply:
makes sense. we'll merge during the continuing refactors